packages/components/src/alignment-matrix-control/index.tsx

```diff
@@ -56,7 +56,7 @@ export function AlignmentMatrixControl( {
 		id
 	);
 
-	const compositeStore = useCompositeStore( {
+	const compositeStore = Composite.useStore( {
```

Comment:
This is actually an interesting one when it comes to naming. Since we're namespacing the compound component under the `Composite` namespace, it felt redundant to call the hook `useCompositeStore`. I therefore opted for `useStore`, expecting consumers to use it as `Composite.useStore`

Reply:
Another consideration to make is whether we think that hooks should be part of the compounding namespace at all.  For example, one inconvenience of having to invoke `Composite.useStore` is that linters recognize hooks based on the fact that they start with `use`